CONTRIBUTING.md

```diff
@@ -399,4 +421,4 @@ will change these into the templates above
 [^3]: Please view the [markdown of this page][contrib-md] for an example of KaTeX math using `$` symbols.
 
 [new-gitbook]: https://www.gitbook.com/
-[contrib-md]: https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/edit/master/CONTRIBUTING.md
+[contrib-md]: https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/edit/master/CONTRIBUTING.md#mathematics-with-tex
```

Comment:
That anchor doesn't work for me. Besides, wouldn't you rather want to link to the raw page?

Reply:
Oh, I had forgot the way github handles that is different from jekyll. And yea, probably will revert this since linking to the whole page works just fine.